CHANGELOG.rst

```diff
@@ -1,6 +1,11 @@
 django-storages change log
 ==========================
 
+UNRELEASED
+**********
+
+* Dropped support for Python 3.3.
```

Comment:
If the maintainers agree to drop support for Django < 1.11 in the next release (along with adding Django 2.0 support) this could be omitted.

Reply:
I'm open to this and don't mind doing any necessary work. @jschneier Let me know how you feel about this proposal.

Reply:
It seems like this tools should follow the Django deprecation rules. In next, release drop support for everything < 1.11 which includes Python 3.3.

Reply:
I agree. This has been done in PR #416.